src/ideabox/policy/content/campaign.py

```diff
@@ -12,6 +12,12 @@ class ICampaign(model.Schema):
         title=_(u"Enable / Disable project submission"), default=False
     )
 
+    emails = schema.TextLine(
+        title=_(u"Mails address"),
+        description=_(u'You can set many mails address separated with ";"'),
```

Comment:
The description should mention the purpose of this field : This email addresses are used to send notification when a new project is proposed by an user.